### PR TITLE
Refactor /home automated tests

### PR DESCRIPTION
**Pull request by mjaskot**
Finished refactoring /home automated tests.

### CODE REVIEW

File: cypress/integration/home/home_spec.js [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+              .should('exist')
+              .and('be.visible')
+          })
+        })
+      })
+    })
+
+    it('"Contact" rendering section', () => {
+      cy.get(`${contact.sectionSelector} > div`).within(() => {
+        Array(0, 1, 2).forEach(index => {
```
Comment (szczepanbarszczowski) [inline cypress/integration/home/home_spec.js]: `[...Array(3)].forEach((_, idx) => ...`

File: cypress/integration/home/home_spec.js [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+              .find(`input`)
+              .should(`have.attr`, `placeholder`, contact.placeholders[index])
+          }
+        })
+      })
+    })
+
+    it('"Contact" empty inputs validation', () => {
+      cy.get(`${contact.sectionSelector} > div`).within(() => {
+        Array(0, 1, 2).forEach(index => {
```
Comment (szczepanbarszczowski) [inline cypress/integration/home/home_spec.js]: `[...Array(3)].forEach((_, idx) => ...`

File: cypress/integration/home/home_spec.js [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+          cy.get(`form`)
+            .find('div')
+            .contains('Required.')
+        })
+      })
+    })
+
+    it('"Contact" length validation', () => {
+      cy.get(`${contact.sectionSelector} > div`).within(() => {
+        Array(0, 1, 2).forEach(index => {
```
Comment (szczepanbarszczowski) [inline cypress/integration/home/home_spec.js]: `[...Array(3)].forEach((_, idx) => ...`

File: cypress/integration/home/data.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+export const clutch = {
+  title: 'CLUTCH',
+  sectionSelector: '[data-cy="clutch"]',
+  rating: ['Rated ', '4.9', '/', '5', ' on'],
+  headers: ['Leading product design', 'agency on Clutch'],
+  cards: {
+    cardSelector: [
+      `[data-cy="clutch"] > div  > div:nth-child(2) > div > div:nth-child(1)`,
```
Comment (galkowskit) [inline cypress/integration/home/data.js]: Why backtick if there is no interpolation inside that string?

Comment (mjaskot) [reply cypress/integration/home/data.js]: @galkowskit My mistake, this selector was copied from one of deleted files where every string was written in backticks. I will correct it.

File: cypress/integration/home/home_spec.js [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+              .should('exist')
+              .and('be.visible')
+          )
+        )
+      )
+    })
+
+    it('"Contact" rendering section', () => {
+      cy.get(`${contact.sectionSelector} > div`).within(() =>
+        [...Array(3)].forEach((_, idx) => {
```
Comment (hypeofpipe) [inline cypress/integration/home/home_spec.js]: You can change if-else to:
cy.get('form)
.children()
.eq(idx)
.find(idx === 2 ? 'textarea' : 'input')
.should('have.attr', 'placeholder', contact.placeholders[idx]

File: cypress/integration/home/home_spec.js [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+            cy.get(`form`)
+              .children()
+              .eq(idx)
+              .find(`textarea`)
+              .should(`have.attr`, `placeholder`, contact.placeholders[idx])
+          } else {
+            cy.get(`form`)
+              .children()
+              .eq(idx)
+              .find(`input`)
```
Comment (hypeofpipe) [inline cypress/integration/home/home_spec.js]: Change all '`' to single apostrophe

File: cypress/integration/home/home_spec.js [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+              .contains(content)
+              .should('exist')
+              .and('be.visible')
+          )
+        )
+      )
+    })
+
+    it('"Contact" rendering section', () => {
+      cy.get(`${contact.sectionSelector} > div`).within(() =>
```
Comment (hypeofpipe) [inline cypress/integration/home/home_spec.js]: You can change if-else to:
cy.get('form)
.children()
.eq(idx)
.find(idx === 2 ? 'textarea' : 'input')
.should('have.attr', 'placeholder', contact.placeholders[idx]

Comment (mjaskot) [reply cypress/integration/home/home_spec.js]: Done, I have also done this in few other places to remove redundant if statements

**Pull request merged**